src/game/SaveLoadScreen.cc

```diff
@@ -56,7 +56,7 @@
 #define SAVE_LOAD_NORMAL_FONT				FONT12ARIAL
 #define SAVE_LOAD_NORMAL_COLOR				2//FONT_MCOLOR_DKWHITE//2//FONT_MCOLOR_WHITE
 #define SAVE_LOAD_NORMAL_SHADOW_COLOR			118//121//118//125
-/*#define SAVE_LOAD_NORMAL_FONT				FONT12ARIAL
+/*#define SAVE_LOAD_NORMAL_FONT			FONT12ARIAL
```

Comment:
this is remaining comment in multi line style, afair, selaux and you advocated only '//' style comments